dips/dip-196.md

```diff
@@ -0,0 +1,55 @@
+---
+dip: 196
+title: Onchain Network Identity
+authors: Greg Nazario (@gregnazario)
+status: Draft
+type: Informational
+created: 07/06/2021
+last_updated: 07/06/2021
+issue: https://github.com/diem/dip/issues/196
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Summary
+
+Diem nodes need the ability to find other nodes in the network easily.  Additionally, the nodes need the ability to find 
+the associated network identity so that the nodes can trust them.  This DIP describes the ability for node owners to
+self register their network identity onchain, and allow other nodes to lookup this data for discovery of other nodes.
```

Comment:
```suggestion
self register their network identity on-chain, and allow other nodes to lookup this data for discovery of other nodes.
```

Reply:
Per David's comment above, the on-chain network identify seems to only be a feature for VFNs.  It might make this DIP more straightforward to focus on that aspect unless there are other concrete use cases you would like to call out.

Reply:
Sure, I can focus on only the VFN usecase